### DISCUSSION TITLE
World Package 1.4.0버전이 업데이트 되었습니다

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#831] [1. Announcements]
안녕하세요.
World Package 1.4.0버전이 업데이트 되어서 안내드립니다.

### 안내
[패키지 업데이트 방법](https://studio.zepeto.me/kr/guides/updating-package)
[1.4.0 릴리즈 노트 보기](https://studio.zepeto.me/kr/guides/world-140)

<hr/>

### 개선된 기능

- 개행 문자 처리 - OSX/Win 메타 파일 변경 이슈 수정
- TypeScript Property 초기 변수 값 적용 안되는 이슈 수정 https://github.com/naverz/zepeto-studio-kor/discussions/409

**Comment by Cha-SeongHun** [comment]
- TypeScript Property 초기 변수 값 적용 안되는 이슈 수정
1.4.0으로 업데이트 후 테스트해보니 위 항목 버그가 그대로 재현됩니다. 확인 부탁드립니다

**Reply by Zepeto-TechEvangelist** [reply]
업데이트 이후 reimport all 하고 테스트 해보시겠어요?
#409 이슈 재현스텝대로 진행시 저는 잘 나오고 있습니다. 

<img width="292" alt="스크린샷 2022-06-03 오후 2 24 47" src="https://user-images.githubusercontent.com/100118650/171791853-6c85738e-fd1e-4185-b24b-daeec9cd47d9.png">

**Reply by Cha-SeongHun** [reply]
Reimport All 이후 다시 테스트해보아도 동일증상 있습니다!

**Reply by Zepeto-TechEvangelist** [reply]
사용하신 코드를 공유해주시면 도움이 될 것 같습니다.

**Reply by Cha-SeongHun** [reply]
테스트에 사용한 컴포넌트의 코드입니다.
targetSeconds 변수를 설정 후 Play를 누르면 프리팹의 값으로 초기화됩니다.

```ts
import { GameObject, Time } from "UnityEngine";
import { ZepetoScriptBehaviour } from "ZEPETO.Script";

export default class ObjectDestroyer extends ZepetoScriptBehaviour {
  public targetSeconds: float;
  public destroyIfDisable: bool = false;

  private elapsedSeconds: float = 0;

  private Update() {
    this.elapsedSeconds += Time.deltaTime;

    if (this.elapsedSeconds >= this.targetSeconds) {
      GameObject.Destroy(this.gameObject);
    }
  }

  private OnDisable() {
    if (this.destroyIfDisable) {
      GameObject.Destroy(this.gameObject);
    }
  }
}

```

**Reply by Zepeto-TechEvangelist** [reply]
일반적인 타입스크립트의 프로퍼티 변수값 이슈는 이번에 포함되었으나
프리팹은 다음 버전 업데이트 또는 핫픽스로 반영될 예정입니다.
감사합니다.
